pyodide-build/pyodide_build/buildpkg.py

```diff
@@ -83,13 +83,17 @@ def close(self):
 def get_bash_runner():
     PYODIDE_ROOT = os.environ["PYODIDE_ROOT"]
     env = {
-        "PATH": os.environ["PATH"],
-        "PYTHONPATH": os.environ["PYTHONPATH"],
-        "PYODIDE_ROOT": PYODIDE_ROOT,
-        "PYTHONINCLUDE": os.environ["PYTHONINCLUDE"],
-        "NUMPY_LIB": os.environ["NUMPY_LIB"],
-        "PYODIDE": "1",
+        key: os.environ[key]
+        for key in [
+            "PATH",
+            "PYTHONPATH",
+            "PYODIDE_ROOT",
+            "PYTHONINCLUDE",
+            "NUMPY_LIB",
+            "PYODIDE_PACKAGE_ABI",
```

Comment:
It turns out the failure was because the fix I contributed upstream to scikit-image checked for the PYODIDE_PACKAGE_ABI variable, however, we deprecated it in favor of the `PYODIDE`. Re-adding that variable should fix the build.

Reply:
It's nice that #2053 made it more explicit which environment variables we are passing down, but it does lead to some pain working out what is going on.

Reply:
Agreed, I like that it's explicit now.